ocaml/xapi/console.ml

```diff
@@ -230,7 +230,19 @@ let real_proxy' ~__context ~vm vnc_port s =
     debug "Proxy exited"
   with exn -> debug "error: %s" (ExnHelper.string_of_exn exn)
 
-let real_proxy __context vm _ _ vnc_port s =
+let respond_console_limit_exceeded ~__context req s =
+  let session_id = Xapi_http.get_session_id req in
+  let user = Db.Session.get_auth_user_name ~__context ~self:session_id in
+  let body =
+    Printf.sprintf
+      "<html><body><h1>Connection Limit Exceeded</h1><p>User '%s': Only 1 \
```

Comment:
Why is this useful? The caller would know itself right?

Reply:
This could create invalid HTML if `user` has characters with special meaning in HTML that would require escaping. 

!https://miro.medium.com/v2/resize:fit:720/format:webp/1*P4nj9fJjSeJ9-c0rwSZqlg.png